app/component/itinerary/navigator/NaviCardExtension.js

```diff
@@ -129,18 +131,66 @@ const NaviCardExtension = ({ legType, leg, nextLeg }, { config }) => {
       </div>
     );
   }
+  if (legType === LEGTYPE.MOVE && nextLeg) {
+    const { headsign, route, start } = nextLeg;
+    const hs = headsign || nextLeg.trip?.tripHeadsign;
+    const remainingDuration = Math.max(
+      Math.ceil((legTime(start) - time) / 60000),
+      0,
+    ); // ms to minutes, >= 0
+    const rt = nextLeg.realtimeState === 'UPDATED';
+    const values = {
+      duration: withRealTime(rt, remainingDuration),
+      legTime: withRealTime(rt, legTimeStr(start)),
+    };
+    const routeMode = getRouteMode(route, config);
+    const iconColor =
+      config.colors.iconColors[`mode-${routeMode}`] ||
+      route.color ||
+      'currentColor';
+    return (
+      <div className="extension">
+        {stopInformation()}
+        <div className="extension-divider" />
+        <div className="wait-leg">
+          <div className="route-info">
+            <div className="expand">
+              <Icon img="navi-expand" className="icon-expand-small" />
+              <div className="wait-duration">
+                <FormattedMessage
+                  id="navileg-arrive-at"
```

Comment:
Just a minor issue, but obviously translation id `navileg-arrive-at` is misleading in thsi context.